INSTALL

```diff
@@ -56,6 +68,11 @@ How to install PBS Pro using the configure script.
   following command:
 
     ./configure --prefix=/opt/pbs --libexecdir=/opt/pbs/libexec
+    
+  For Debian systems you should run the following command:
+
+    CFLAGS='-I/usr/include/tcl8.6' \
+      LDFLAGS='-L/usr/lib/x86_64-linux-gnu -ltclstub8.6' ./configure
```

Comment:
m4/with_tcl.m4 should be modified so these special directives aren't required under Debian. We had no choice with lib exec for openSUSE, but we should try to avoid extra parameters to configure wherever possible. If it can't be handled by modifying with_tcl.m4, then the CFLAGS and LDFLAGS variables should be passed as arguments to configure (e.g. ./configure CFLAGS= ...). The reason for this is because they will be captured as arguments in the config.status that gets generated.